### PR TITLE
Registry replacer: add pr creation functionality

### PR DESCRIPTION
**Pull request by alvaroaleman**
This PR adds pr-creating abilities to the registry replacer

### CODE REVIEW

File: cmd/registry-replacer/main.go [hunk 1]
```diff
@@ -14,24 +16,73 @@ import (
...
+	selfApprove    bool
+	flagutil.GitHubOptions
+}
+
+func gatherOptions() (*options, error) {
+	o := &options{}
+	flag.StringVar(&o.configDir, "config-dir", "", "The directory with the ci-operator configs")
+	flag.BoolVar(&o.createPR, "create-pr", false, "If the tool should automatically create a PR. Requires --token-file")
+	flag.StringVar(&o.githubUserName, "github-user-name", "openshift-bot", "Name of the github user. Required when --create-pr is set. Does nothing otherwise")
+	flag.BoolVar(&o.selfApprove, "self-approve", false, "If the bot should self-approve its PR.")
```
Comment (stevekuznetsov) [inline cmd/registry-replacer/main.go:43]: Do we want a common options struct to hold these and wrap the interaction with the library?

Comment (alvaroaleman) [reply cmd/registry-replacer/main.go:43]: not really worth it at this point, this is about to options, a boolean and a string, I don't think we gain much from sharing that

File: images/registry-replacer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM centos:7
+LABEL maintainer="muller@redhat.com"
```
Comment (petr-muller) [inline images/registry-replacer/Dockerfile:2]: :trollface:

File: images/registry-replacer/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM centos:7
```
Comment (petr-muller) [inline images/registry-replacer/Dockerfile:1]: Care to bump to 8 given you bumped the applyconfig one?

Comment (alvaroaleman) [reply images/registry-replacer/Dockerfile]: Done

**Pull request merged**